### ISSUE TITLE
Support for alignment in NSAttributedString

### ISSUE THREAD

**Issue by viralsavaniIM** [huemae/SuggestionsKit#3]
Hey, @huemae big thanks for writing this neat library.

I have been playing around with it for a few days and ran into this issue. So I'm trying to left-align the text shown in the bubble. But can't get it working using 
Here is the snippet of the code I'm using `NSAttributedString`.

```
let paragraphStyleLeftAlign = NSMutableParagraphStyle()
paragraphStyleLeftAlign.alignment = .left

let exampleAttrString1 = NSAttributedString(string: "frame suggestion #1 The quick brown fox jumps over the lazy dog, long text to test left align",
                                            attributes: [.font: UIFont.systemFont(ofSize: 25, weight: .thin),
                                                        .foregroundColor: UIColor.lightGray,
                                                        .paragraphStyle: paragraphStyleLeftAlign])
```

#### Here is the output
<img width="443" alt="image" src="https://user-images.githubusercontent.com/22685974/101717126-90be0180-3a53-11eb-9005-be9bd7895860.png">

As you can see the entire text is not aligned correctly.

Any thoughts as to what might be going on? Or if this kind of `NSAttributedString` configuration would not be supported?

**Comment by AlphanumericCharactersOrSingleHyphenz** [comment]
@viralsavaniIM thanks for report! Support of this parameter will be added as soon as possible. (At this weekend)

**Comment by AlphanumericCharactersOrSingleHyphenz** [comment]
@viralsavaniIM I'am glad to announce that this functionality was implemented just now in version 1.2.8.
 You can use example project to test it.
 Please leave your feedback on how it works and is everything ok

**Comment by viralsavaniIM** [comment]
Hey, @huemae nice work. Seems like `1.2.8` fixed it. Thanks for the quick turn around time. I have tested it and looks alright.

A minor suggestion in the example code
https://github.com/huemae/SuggestionsKit/blob/a8ec53c37f215e0bb77026996e6f0ed8c3023c82/Example/SuggestionsKit-Example/ViewController.swift#L86-L87

The variable says `paragraphStyleLeftAlign` but the alignment is pointing to `.center`.
